tests/union/in3a.fsm

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright 2008-2017 Katherine Flavel
```

Comment:
Wrong author! (also for minimise.c now!)

Reply:
Test header copyright fixed in 146b4f8c.

What should I do for minimise.c? While I substantially rewrote it, it's been part of the project for a while.